assets/js/components/notifications/cta.js

```diff
@@ -39,10 +40,12 @@ class CTA extends Component {
 		} = this.props;
 
 		return (
-			<div className={ `
```

Comment:
Very nice getting rid of all of these multiline template string `className` values. Thanks! 👍